### DISCUSSION TITLE
关于一天学习时长

### DISCUSSION THREAD

**Discussion by huangxu2023** [RustCoreBoys/RustLearning#31] [Polls]
时间宝贵，每个人都有自己的课业工作。想调查一下大家每天能够花在学习本课程的时间。

**Comment by huangxu2023** [comment]
测试

**Comment by qpc0422** [comment]
刚肝完mit6.s081,目前在写cmu15445,所以只能每天抽一部分时间来写这个rust,暂时想着每天花大概2h的时间在上面。当然还是随心而学，并不会太在意时间。

**Reply by huangxu2023** [reply]
牛哇牛哇，加油💪🏻
